.travis.yml

```diff
@@ -14,19 +14,47 @@
 
 dist: trusty
 sudo: required
-lang: cpp
+language: cpp
 
-compiler:
-  - clang
-  - gcc
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
```

Comment:
clang on recent versions of Linux (e.g. Ubuntu 17.04, Fedora 27) is triggering warnings in gRPC and/or protobuf.  I am trying to get us coverage and avoid some problems at the same time.

Reply:
That makes sense and sounds good. Would you mind adding a comment to this effect to the file so that it's also clear to future readers, who aren't going to look at this comment thread? 

We should probably try to resolve that issue at some point, whether by tuning Clang flags, or fixing gRPC or protobuf, or something else, and this will keep reminding us.

Reply:
Created #19 to track this, added the comment and also created one clang+Linux build.